azure-webapp-maven-plugin/README.md

```diff
@@ -1,186 +1,177 @@
+        <plugins>
+            <plugin>
+                <groupId>com.microsoft.azure</groupId>
+                <artifactId>azure-webapp-maven-plugin</artifactId>
+                <!-- check Maven Central for the latest version -->
+                <version>1.5.3</version>
+                <configuration>
+                    <schemaVersion>v2</schemaVersion>
+                    <resourceGroup>${RESOURCEGROUP_NAME}</resourceGroup>
+                    <appName>${WEBAPP_NAME}</appName>
+                    <runtime>
+                        <os>linux</os>
+                        <javaVersion>jre8</javaVersion>
+                        <webContainer>tomcat 8.5</webContainer>
+                    </runtime>
+                    <deployment>
+                        <resources>
+                            <resource>
+                                <directory>${project.basedir}/target</directory>
+                                <includes>
+                                    <include>*.war</include>
+                                </includes>
+                            </resource>
+                        </resources>
+                    </deployment>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>      
     ```
-   
-3. Use the following commands to deploy your project to Azure App Service. 
-
+3. Use the following commands to deploy your project to Azure App Service.
     ```
     $ mvn azure-webapp:deploy
     ```
 
-<a name="samples"></a>
-## Samples	
-A few typical usages of Maven Plugin for Azure App Service Web Apps are listed at [Web App Samples](../docs/web-app-samples.md). You can choose one to quickly get started.
-
 <a name="goals"></a>
 ## Goals
 
-The Maven Plugin for Azure App Service Web Apps has only one goal: `azure-webapp:deploy`. 
+#### `azure-webapp:config`
+-  A plugin goal providing a great experience for managing the configuration of [azure-webapp:deploy](#azure-webappdeploy), it will rewrite your pom configuration based on your input. Details about the configuration could be found in the following.
+-  The plugin goal will divide the configuration into 3 parts.
+   1. Application
+        
+        You can configure appName, resource group, region, pricing tier at this part.
+   2. Runtime
+        
+        You can configure operating system and runtime stack at this part. It should be noted that you can not change the operating system after you deploy it.
+   3. DeploymentSlot
+        
+        You can configure deployment slot at this part.
+
+#### `azure-webapp:deploy`
+- Deploy artifacts or docker container image to an Azure Web App based on your configuration.<br>If the specified Web App does not exist, it will be created.
+- Configuration:
+    Common configurations of all Maven Plugins for Azure can be found [here](../docs/common-configuration.md).
+
+    This maven plugin supports two kinds of configurations V2 and V1 (deprecated). Specify the configuration
```

Comment:
Add the schemaVersion to the table below.

Reply:
Meanwhile, this sentence can be a note in the doc. For example, we can put it after the table and write it as:

> Using v2 schema is recommended. Please click [here]() if you want to find more details about v1 schema (deprecated)

Reply:
I think do not need to add schema version to the table, the table will seem too long.